src/mixer/samplerbank.cpp

```diff
@@ -2,16 +2,28 @@
 
 #include <QFileDialog>
 #include <QMessageBox>
+#include <QStandardPaths>
 
 #include "control/controlpushbutton.h"
 #include "mixer/playermanager.h"
 #include "mixer/sampler.h"
 #include "moc_samplerbank.cpp"
 #include "track/track.h"
 #include "util/assert.h"
+#include "util/file.h"
 
-SamplerBank::SamplerBank(PlayerManager* pPlayerManager)
+namespace {
+
+const ConfigKey kConfigkeyLastImportExportDirectory(
+        "[Samplers]", "last_import_export_directory");
+const QString kSamplerFileType = QT_TR_NOOP("Mixxx Sampler Banks (*.xml)");
```

Comment:
```suggestion
const char kSamplerFileType[] = QT_TR_NOOP("Mixxx Sampler Banks (*.xml)");
```

Reply:
Thanks!

Reply:
can explain why a `char[]` is preferred here? `QT_TR_NOOP` is used nowhere else in mixxx so explaining why its used here definitively deserves explanation in the code IMO.

Reply:
_ping_
Shall we add a comment, then merge?

Reply:
Yes please

Reply:
hehe, this was meant for @daschuer

Reply:
QT_TR_NOOP should always used with char[] or char*, because the result will be used as key for fetching the translated string in tr() 

https://doc.qt.io/qt-5/qobject.html#tr

Using QString will convert from char[] and back to char*